vaadin-menu-bar-flow-parent/vaadin-menu-bar-flow-integration-tests/src/test/java/com/vaadin/flow/component/menubar/tests/MenuBarPageIT.java

```diff
@@ -559,6 +560,33 @@ public void subMenuItem_classNamesAreToggleWithSet_classNamesAreToggled() {
                 MenuBarTestPage.SUB_ITEM_FIRST_CLASS_NAME);
     }
 
+    @Test
+    public void menuItemWithClassNameInOverflow_changeClassName_classNameIsChanged() {
+        click("set-width");
+        click("set-item2-class-name");
+        waitForResizeObserver();
+        menuBar.getOverflowButton().click();
+        click("change-item2-class-name");
+        menuBar.getOverflowButton().click();
+        TestBenchElement menuItem = menuBar.getSubMenuItems().get(0);
+        Assert.assertEquals(menuItem.getAttribute("class"),
+                MenuBarTestPage.MENU_ITEM_SECOND_CLASS_NAME);
+    }
+
+    @Test
+    public void menuItemWithClassNameInOverflow_removeClassName_classNameIsRemoved() {
+        click("set-width");
+        click("set-item2-class-name");
+        waitForResizeObserver();
+        menuBar.getOverflowButton().click();
+        click("remove-item2-class-name");
+        menuBar.getOverflowButton().click();
+        TestBenchElement menuItem = menuBar.getSubMenuItems().get(0);
+
+        Assert.assertTrue(
+                Strings.isNullOrEmpty(menuItem.getAttribute("class")));
```

Comment:
nit: `Strings.isNullOrEmpty` is not used anywhere else in the codebase. Is there some more commonly used alternative?

Reply:
Changed to check if the class attribute doesn't contain the value set previously.